src/redux/theme/theme.reducer.js

```diff
@@ -11,10 +11,10 @@ const themeState = (state = initialState, action = {}) => {
       darkMode: action.payload
     };
 
-  case 'SET_DRAWER_STATUS':
+  case 'SET_SIDE_MENU_STATUS':
```

Comment:
Please, move action type to file `{name}.types.js` like in main client to use constants instead of plain strings

Reply:
You're right! Moved!